.travis.yml

```diff
@@ -38,8 +44,8 @@ jobs:
     # we recommend new addons test the current and previous LTS
     # as well as latest stable release (bonus points to beta/canary)
     - stage: "Additional Tests"
-      env: EMBER_TRY_SCENARIO=ember-lts-2.16
```

Comment:
@Exelord it looks like you dropped support for Ember 2.16 here. that does look like a breaking change to me.

Reply:
It's not dropped. It's not tested against anymore, which means the support for 2.16 ended on v2.0 - which should be marked in the readme.

I only applied changes from new ember update blueprint, which doesn't change anything for end users yet. What do you suggest based on this discussion? To release a minor version on every ember upgrade?

Reply:
> To release a minor version on every ember upgrade?

to release a major version whenever support for older Ember.js, Ember CLI or Node.js versions is dropped

Reply:
Sorry I meant major of course

Hmm...so why eg ember-cli is not releasing a new major version if it's dropping support for node 6?

I can agree, this can be a good rule, but it's bearly respected by anyone.

Reply:
because in https://emberjs.com/blog/2016/09/07/ember-node-lts-support.html the CLI team officially declared its support strategy for Node.js releases. the reasoning for CLI is that it is more important to stay on the same version as Ember.js itself, but I personally would still prefer to have bumped the version there too.

Reply:
imho dropping tests for a certain ember version does not necessarily mean a breaking change. If the api changes due to supporting different ember versions, that would definitely be a breaking change.  However, in this case no api has changed. Semantic versioning also does not provide any guarantee regarding test coverage as far as I know. But certainly there are finer points, that can be argued about.
 As @Exelord pointed out, 2.16 support has been dropped with the 2.0 release - which clearly has been a major release. This sounds all good to me, but that's just my personal opinion on that matter.

Reply:
> On the other hand, if everyone would start creating major releases for things like dependency upgrades or dropping support for a certain version (which does not mean it no longer works), we would lose an important feature of semantic versioning: Being able to figure out when we need to take action and adapt our code, just by looking at version numbers.

but that is exactly the point, if you're still using Node 6 and some dependency drops support for it without a new major release then you're in trouble now.

> As @Exelord pointed out, 2.16 support has been dropped with the 2.0 release

I don't see that mentioned in https://github.com/davewasmer/ember-cli-favicon/blob/master/CHANGELOG.md#v200-2018-11-19